packages/edit-site/src/components/sidebar-navigation-screen-library/category-item.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import SidebarNavigationItem from '../sidebar-navigation-item';
+import { useLink } from '../routes/link';
+
+export default function CategoryItem( {
+	count,
+	icon,
+	id,
+	isActive,
+	label,
+	type,
+} ) {
+	const linkInfo = useLink(
+		{
+			path: '/library',
+			categoryType: type,
+			categoryId: id,
+		},
+		{
+			// Keep record for where we came from in a state so we can
```

Comment:
```suggestion
			// Keep record of where we came from in a state so we can
```

Reply:
Fix up the wording here as there were more issues than `for`/`of`: https://github.com/WordPress/gutenberg/pull/51078/commits/43b4bbefd8ae1a056d80931bbdc0c0438191f304